dmpy/client.py

```diff
@@ -0,0 +1,147 @@
+import hashlib
+import json
+import logging
+from datetime import datetime
+from pathlib import Path
+
+import requests
+from dotenv import get_key, load_dotenv, set_key
+from requests_toolbelt.multipart.encoder import (
+    MultipartEncoder,
+    MultipartEncoderMonitor,
+)
+
+from .core.payloads import FileUploadPayload
+from .core.utils import read_text_resource
+
+log = logging.getLogger(__name__)
+
+
+class Dmpy:
+    def __init__(self):
+        self.env = ".dmpy.env"
+        self.url = get_key(self.env, "DMP_URL")
+        load_dotenv(self.env)
```

Comment:
Our of curiosity, why don't you load in all the .envs as with `self.url`? Would make subsequent calls such as `get_key()` a bit simpler. Or are your trying to exclude it from any inspectable entity?

Reply:
No reason -- likely an oversight as all environemtnal variable usage is in `access_token` so wanted to keep them in one place. I agree with you and will update it 👍🏼